### PR TITLE
test: add tests for PolicyAuthorizeNV

### PR DESCRIPTION
**Pull request by tstruk**

### CODE REVIEW

File: test/integration/policy-authorizeNV.int.c [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    LOG_INFO("Tss2_Sys_NV_UndefineSpace for NV index 0x%" PRIx32 " success!",
+             nv_index);
+
+
+    /* Call encrypt using the key object using the policy session again.
+     * This should fail because the NV index is destroyed */
+    cmd_auth.auths[0].sessionHandle = session_handle;
+    memset(data_out.buffer, '\0', TPM2_MAX_DIGEST_BUFFER);
+    LOG_INFO("Calling EncryptDecrypt again with policy session after destroing the NV index");
+    LOG_INFO("This shoudl fail with RC_POLICY_FAIL");
```
Comment (AndreasFuchsSIT) [inline test/integration/policy-authorizeNV.int.c]: Typo... Also also maybe merge into 1 LOG_INFO

File: test/integration/policy-authorizeNV.int.c [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                                   nv_index,
+                                   &cmd_auth,
+                                   &rsp_auth);
+    if (rc != TSS2_RC_SUCCESS) {
+        LOG_ERROR("Tss2_Sys_NV_UndefineSpace failed: 0x%" PRIx32, rc);
+        exit(1);
+    }
+    LOG_INFO("Tss2_Sys_NV_UndefineSpace for NV index 0x%" PRIx32 " success!",
+             nv_index);
+
```
Comment (AndreasFuchsSIT) [inline test/integration/policy-authorizeNV.int.c]: Remove extra line ?

Comment (tstruk) [reply test/integration/policy-authorizeNV.int.c]: Done

File: test/integration/policy-authorizeNV.int.c (hunk 1 above)
Comment (tstruk) [reply test/integration/policy-authorizeNV.int.c]: Done

**Pull request merged**